### PR TITLE
Use textarea in variable values

### PR DESCRIPTION
**Pull request by fviale**
When entering long values in variables, as the input element width is not resizable, it is complicated to view the value entered or spot mistakes.

This changes the input field used for variable-like values (Job and Task variables, Generic information, and job submission) from text to textarea, allowing to resize the field and thus better view the entered value.

Additionally:
 - make all textarea resizable vertically only (resizing horizontally creates issues on the studio).
 - change the label "Number of Automatic Restarts", which is incorrect, to "Number of Execution Attempts"

### CODE REVIEW

File: app/styles/studio.css [hunk 1]
```diff
@@ -517,10 +517,16 @@ ANALOGIC SCHEME
     min-height: 2em !important;
 }
 
+.textareavalues {
+    min-height: 1em !important;
+    min-width: unset;
```
Comment (chloegugli) [inline app/styles/studio.css]: Why did you do ' min-width: unset;' ?

Comment (fviale) [reply app/styles/studio.css]: otherwise it inherits from min-width: 90% which does not align with other fields in the modal form

Comment (fviale) [reply app/styles/studio.css]: but maybe there is a better approach than unset

Comment (chloegugli) [reply app/styles/studio.css]: Another solution would be to look at the CSS of the other fields. But this seems OK too.

Comment (fviale) [reply app/styles/studio.css]: ok, I will leave it like this then.

**Pull request merged**